website/docs/reference/define-properties.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Define properties
+sidebar_label: Define properties
+intro_text: "Learn how to define properties for your resources in a properties.yml file"
```

Comment:
Same question as above for best practices for intro text - Should our intro text be more descriptive than the descriptions?